src/views/Timer.vue

```diff
@@ -0,0 +1,105 @@
+<template lang="pug">
+div
+  h2 Timer
+  p
+    | Using bucket: {{bucket_id}}
+
+  b-alert(show)
+    | This is an early experiment, an important missing feature is the ability to set start/end of timers manually.
+
+  b-input-group(prepend="New timer", size="lg")
+    b-input(v-model="label" placeholder="Label")
+    b-input-group-append
+      b-button(@click="startTimer(label)", variant="success")
+        icon(name="play")
+        | Start
+
+  hr
+
+  div.row
+    div.col-md-6
+      h3 Running timers
+      div(v-for="e in runningTimers")
+        timer-entry(:event="e", :bucket_id="bucket_id", :now="now", @delete="deleteTimer(e)")
+        hr(style="margin: 0")
+
+    div.col-md-6
+      h3 Stopped timers
+      div(v-for="e in stoppedTimers")
+        timer-entry(:event="e", :bucket_id="bucket_id", :now="now", @delete="deleteTimer(e)", @new="startTimer(e.data.label)")
+        hr(style="margin: 0")
+</template>
+
+<style scoped lang="scss">
+.btn {
+  margin-right: 0.5em;
+
+  .fa-icon {
+    margin-left: 0;
+    margin-right: 0.5em;
+  }
+}
+</style>
+
+<script>
+import _ from 'lodash';
+import moment from 'moment';
+
+import TimerEntry from '../components/TimerEntry.vue';
+import 'vue-awesome/icons/play';
+import 'vue-awesome/icons/trash';
+
+export default {
+  name: "Timer",
+  components: {
+    "timer-entry": TimerEntry
+  },
+  mounted: function() {
+    // TODO: List all possible timer buckets
+    //this.getBuckets();
+
+    // Create default timer bucket
+    this.$aw.ensureBucket(this.bucket_id, "timer", "unknown");
```

Comment:
We shouldn't use "timer" as a eventtype, we are trying to use eventtypes in the format of "os.window.current" etc.

Reply:
It's stuff like this that I don't want to be merged early, because once that's in and people start using it we can't change it

Reply:
Do you have a better suggestion for the event type? I can't really come up with anything.

Reply:
general.timer.stopwatch or something similar would be my suggestion
Timers are not really part of any os/web/whatever so general seems fine, it's a type of timer and the specific type of timer is a stopwatch.

Reply:
@karvla agreed. I went with `general.stopwatch`. See #130